docs/topics/http/middleware.txt

```diff
@@ -90,6 +90,17 @@ caveats:
 * Unlike the ``__call__()`` method which is called once per request,
   ``__init__()`` is called only *once*, when the Web server starts.
 
+``__call__(request)``
+---------------------
+
+Middleware factories are made callable by implementing the ``__call__()``
+method and must accept a ``request`` argument and return a response. The
+response can be retrieved by calling its ``get_response``. If the
+``__call__()`` method returns a response without every calling its
```

Comment:
"without ever"

Reply:
Thanks @jacobtylerwalls for your suggestion! I've applied it in 5f89049.